packages/venia-concept/src/components/CategoryList/categoryList.css

```diff
@@ -0,0 +1,48 @@
+.root {
+    padding: 1rem;
+}
+
+.header {
+    margin-bottom: 2rem;
+    text-align: center;
+}
+
+.title {
+    text-transform: uppercase;
+}
+
+.content {
+    display: grid;
+    grid-gap: 3rem 1rem;
+    grid-template-columns: repeat(auto-fit, 6rem);
```

Comment:
@jimbo would give better advice here, but I think you might want to use a `fr` unit instead of `6rem`, if you want a predictable number of items in each row.

Reply:
We have mobile, tablet, desktop view.
My idea was to have fixed width of items + margin between them.
Center aligned, no matter which device we are talking about.

![screen shot 2018-07-26 at 17 39 51](https://user-images.githubusercontent.com/22525219/43272760-1136dc1a-90fb-11e8-822a-82e59f21382e.png)
![screen shot 2018-07-26 at 17 40 23](https://user-images.githubusercontent.com/22525219/43272761-11633d46-90fb-11e8-8179-d2c19dc7f0be.png)

If you have some other solution please let me know.

Reply:
The approach used here achieves the design they're aiming for, so it looks good to me. Thanks for embracing grid. 👍